ArcGISRuntimeSDKQt_QMLSamples/Maps/MapReferenceScale/MapReferenceScale.qml

```diff
@@ -0,0 +1,253 @@
+// [WriteFile Name=MapReferenceScale, Category=Maps]
+// [Legal]
+// Copyright 2018 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import Esri.ArcGISRuntime 100.5
+import QtQuick.Controls 2.2
+import Esri.ArcGISExtras 1.1
+import QtQuick.Layouts 1.12
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    property var referenceScales: [500000,250000,100000,50000]
+    property string webMapId: "3953413f3bd34e53a42bf70f2937a408"
+    property LayerListModel featureLayerModel
+
+    MapView{
+        id: mapView
+        anchors.fill: parent
+
+        // Create a Map from a Portal Item
+        Map {
+            id: map
+
+            PortalItem {
+                id: mapPortalItem
+                portal: portal
+                itemId: webMapId
+                onErrorChanged: {
+                        console.log("Error in portal item: " + error.message + "(" + error.additionalMessage + ")");
```

Comment:
optional, but Qt extended the JS String to have a .arg() to make string concatenation easier - https://doc.qt.io/qt-5/qml-qtqml-string.html#arg-method